### DISCUSSION TITLE
CustomField duplicate - remove useless annoying "copy"

### DISCUSSION THREAD

**Discussion by justdudeone** [pods-framework/pods#6798] [Ideas]
When admin set Pod CustomFields and click duplicate - new CustomField appear with added "copy" word at the end of Name and Slug. It's really useless and very annoying when you add for example 50 duplicated fields and need to remove "copy" from each of them. Anyway you have check for unique slug thus admin will be warned. Just remove adding "copy".

**Edit note:** Removed unrelated/unnecessary text, @sc0ttkclark

**Comment by sc0ttkclark** [comment]
The use case here for the Duplicate Field feature isn't for doing it 50 times, it's meant to duplicate a few fields. Many people don't recognize which field is which without a change to the label. This ensures the most people can understand that their field was duplicated and which one that new one is. They can go and edit from there.

In the future, maybe we can find a way to make this optional through a filter.

**Comment by justdudeone** [comment]
Such your manner of categorical restriction to the active users of "how users must use your Pods, how users must duplicate fields, how much times users must duplicate fields, etc" - are really don't have any sense. If any SW have some function - user can use it, that's all. User can use it 1 time, or 100500 times as he needs it. Developer's task is to make ANY using of ANY existed function comfortable for user. For example I need to add 50 similar CustomFields to Pod with special options. What is better: (a) to add 1by1 and setup each field manually? or (b) just setup 1 field and duplicate it 50 times? Obvious that option (b) are better and faster and more more convenient. But then I face an inconvenience to delete your "copy" at Name+Slug 50*2=100 times!!! What a useless and nonsense time wasting?! Just remove this add "copy", that's all. Your left-side fields status indicators are really very nice and enough for this purpose. Or you can add 1 more color which will show that duplicated field need editing, for example when Slug is the same for >1 CsutomField at this Pod.

**Comment by JoryHogeveen** [comment]
If you duplicate a field you always need to change the label and name of the field right? Why is it such an issue that Pods adds "copy" to make the user knows which field is the duplicate?
Plugins like Duplicate Post do the same. Even OS do the same if you duplicate files.

As Scott mentioned, we could add a filter in the future that omits the "Copy" postfix, however, since each field needs to be unique, we always need to add something to the label and name to make sure it's different from the original field.

**Comment by justdudeone** [comment]
Any enough smart admin before duplicate field will make the first comfortable "dummy-template"... like this:
1) Firstly create field with name "Section Two: Question " and slug "section_two_question_". Set all field settings. 
2) Duplicate field 50 times.
3) Just add at the end number 01-50, like name "Section Two: Question 01" and slug "section_two_question_01" etc.
Only with your Pods duplicating function admin forced to waste time to delete the "copy" word for 100 times.

To resolve this you must remove "copy" words and also you can replace it by: 
- or add 1 more side indicator color for duplicated fields like yellow or pinky etc;
- or add standalone indicator "copy", but not a name or slug fields;
- or even add the "fields templates" at right column and possibility to add them easily by 1 click, thus it will work as "insert dummy-template function for many fields with similar settings" (to add any field to the"dummy-templates" list you can add one more button on each field block).

**Comment by justdudeone** [comment]
If at old interface each click "Duplicate" - instantly creates new custom field copy with ".... (copy)" words...
At newer beta version interface when click on "Duplicate" it opens the new window with the same "copy" words - EVEN WHEN THIS FIELD DOES NOT SAVED or CREATED!!! and EVEN WHEN dialog closing "Save field" button have check for filed name uniqueness - those "copy" words are totally useless and have no any sense!!!
Remove hose annoying "copy" words on fields duplicate!
